src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/ClientWebSocketOptions.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.WebSockets
+{
+    public sealed class ClientWebSocketOptions
+    {
+        private bool _isReadOnly; // After ConnectAsync is called the options cannot be modified.
+        internal List<string>? _requestedSubProtocols;
```

Comment:
Why is it internal?

Reply:
not sure to tell you the truth.  I followed the same as here:  https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs#L27

Can change this one as well as the one in master as well.